src/utils/getComponents/getComponents.js

```diff
@@ -65,8 +65,9 @@ export default function getComponents({ areas, templates }: TAreasList) {
       ? AreaComponent
       : withPlaceholder(AreaComponent, areaParams)
 
-    return Object.assign({}, components, {
+    return {
+      ...components,
```

Comment:
ES6 for the win 👍

Reply:
Turned out we are already using rest-spread support through Babel. No need for explicit `Object.assign` :)